tools/pipelines/test-real-service.yml

```diff
@@ -87,7 +97,10 @@ stages:
       artifactBuildId: $(resources.pipeline.client.runID)
       testCommand: test:realsvc:routerlicious:report
       continueOnError: true
-      r11sSelfSignedCertSecureFile: wu2-tls-certificate.pem
+      ${{ if eq(parameters.useR11sPpeEnvironment, true) }}:
+        r11sSelfSignedCertSecureFile: wu2-ppe-tls-certificate.pem
```

Comment:
Could it be possible that when we move to the production stage, the setting is set to "true," and we are using the wrong certificate, leading to an authentication failure?

Reply:
Not really, the parameter controls both the certificate and the endpoint that gets used, so those two should always be in sync. Also, the only way to change the parameter is by manually checking a box during a manual pipeline run, and the default value for the parameter maintains the exact same behavior we have today, so I think it's overall low risk.